contents/newsletter.md

```diff
@@ -4,24 +4,29 @@ sidebar: Docs
 showTitle: true
 ---
 
-PostHog publishes a weekly newsletter.
+![PostHog Newsletter](images/docs/newsletter/newshogs.png)
 
-It contains our latest content:
-
-* Feature Updates
-* Security Updates
-* Blog Posts
-* PostHog News
-
-<link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css" />
-<div id="mc_embed_signup">
-<form action="https://posthog.us19.list-manage.com/subscribe/post?u=292207b434c26e77b45153b96&amp;id=97194afa0a" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
-    <div id="mc_embed_signup_scroll">
-	    <label for="mce-EMAIL">Stay up to date with PostHog</label>
-	    <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required />
-        <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text"   name="b_292207b434c26e77b45153b96_97194afa0a" tabindex="-1" value="" /></div>
-        <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button" />
-        </div>
+<Section
+    divider={false}
+    title="HogMail is a fortnightly newsletter that’s about helping you make better products."
```

Comment:
As an American, I'm not quite sure what "fortnightly" means! =]

Reply:
Good to know! Had no idea you guys didn't use the term. 

It means every two weeks, or bi-weekly. But bi-weekly is super confusing because it can mean twice a week or every two weeks!

Maybe for now we just drop the timeframe thing entirely?